### PR TITLE
Android crash reports on production

### PR DESCRIPTION
**Pull request by AlexeyBarabash**
Uplift of https://github.com/brave/brave-variations/pull/394
Increase rate of PostFREFixMetricsReporting/MetricsReporting up to 100%

### CODE REVIEW

**Review by bsclifton — APPROVED** [review]
++

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `staging` via https://github.com/brave/brave-variations/pull/394#issuecomment-1284900428.

**Pull request merged**
